scripts/zones/Empyreal_Paradox/bcnms/Apocalypse_Nigh.lua

```diff
@@ -0,0 +1,41 @@
+-----------------------------------
```

Comment:
Naming convention for battlefield luas is: `all_lower_case.lua`

Note: to fix this, don't use Windows to rename the file.  Instead, use your Git client to do it, so Git knows about the rename.  You use TortoiseGit, so you could either:

* right click the file > TortoiseGit > rename

or if you prefer bash command line,

* `git mv <old filename> <new filename>`